packages/api/lib/testUtils.js

```diff
@@ -537,7 +536,6 @@ const createPdrTestRecords = async (context, pdrParams = {}) => {
   });
 
   const insertPgRecord = await translateApiPdrToPostgresPdr(originalPdr, knex);
```

Comment:
Nitpick: but can we refactor this function to use `fakePdrRecordFactory` so it uses a PG style record instead of translating API -> PG. Then also translating `originalPgRecord` for writing to ES via `translatePostgresToApiPdr`. I'd like to remove our reliance on API style records/helpers as we move away from Dynamo.

But this isn't strictly necessary.

Reply:
addressed the comments

Reply:
Thanks!